include/swift/AST/DiagnosticsParse.def

```diff
@@ -748,6 +748,8 @@ ERROR(expected_dynamic_func_attr,none,
 ERROR(async_after_throws,none,
       "'async' must precede %select{'throws'|'rethrows'}0", (bool))
 ERROR(async_init,none, "initializer cannot be marked 'async'", ())
+ERROR(duplicate_effect_specifier,none,
+      "unexpected '%0' specifier", (StringRef))
```

Comment:
Nit; maybe instead of saying that it's an unexpected specifier, that it's a duplicate specifier? "Unexpected" specifier isn't immediately actionable.

Reply:
Adjusted the message.